src/base/static/js/views/layer-view.js

```diff
@@ -42,22 +77,32 @@
 
       // Don't draw new places. They are shown by the centerpoint in the app view
       if (!this.model.isNew() && this.isPublishable()) {
+        var geom,
+            styleRule = {},
+            zoomRule = {},
+            styleRuleContext = _.extend({},
+              this.model.toJSON(),
+              {map: {zoom: this.map.getZoom()}},
+              {layer: {focused: this.isFocused}});
+
+        for (var i = 0; i < this.styleRules.length; i++) {
```

Comment:
Re-add the comment here: `// Determine the style rule to use based on the model data and the map state`